config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
+
+  resources :classrooms do
```

Comment:
add `only: [:show]` here and to work groups until we have more actions in the controller

Reply:
Got it! It's changed now.